README.md

```diff
@@ -1163,6 +1163,7 @@ _See also: [awesome-sysadmin/IT Asset Management](https://github.com/awesome-fos
 - [Galette](https://galette.eu/dc/) - Galette is a membership management web application towards non profit organizations. ([Source Code](https://git.tuxfamily.org/galette/galette.git/)) `GPL-3.0` `PHP`
 - [GRR](https://grr.devome.com/?lang=en) - Assets management and booking for small/medium companies. ([Source Code](https://github.com/JeromeDevome/GRR)) `GPL-2.0` `PHP`
 - [Hospital Run](https://hospitalrun.io/) - Hospital Run is offline enabled hospital management software. ([Demo](https://hospitalrun.io/demo/), [Source Code](https://github.com/HospitalRun/hospitalrun-server)) `GPL-3.0` `Nodejs`
+- [Hub20](https://hub20.io/) - A self-hosted payment processor for Ethereum and ERC20 Tokens, integrates with [Raiden Network](https://raiden.network) for instant and cheap transfers. ([Documentation](https://docs.hub20.io/), [Source Code](https://gitlab.com/mushroomlabs/hub20/), [Demo](https://app.hub20.io) - connect to `https:/demo.hub20.io` and register for an account)`AGPL-3.0` `Docker/Python`
```

Comment:
- please remove the inline link to "raiden network"
- please remove the "documentation" link
- if the demo login process is not straightforward please remove the demo link

Reply:
I'm amending the commit now with the requested changes.